dotnet/src/Connectors/Connectors.Anthropic/Models/AnthropicChatMessageContent.cs

```diff
@@ -1,95 +1,44 @@
 
-using System.Collections.Generic;
-using System.Linq;
 using System.Text;
+using System.Text.Json.Serialization;
 using Microsoft.SemanticKernel.ChatCompletion;
-using Microsoft.SemanticKernel.Connectors.Anthropic.Core;
 
 namespace Microsoft.SemanticKernel.Connectors.Anthropic;
 
 /// <summary>
-/// Claude specialized chat message content
+/// Anthropic specialized chat message content
 /// </summary>
 public sealed class AnthropicChatMessageContent : ChatMessageContent
 {
     /// <summary>
-    /// Initializes a new instance of the <see cref="AnthropicChatMessageContent"/> class.
+    /// Creates a new instance of the <see cref="AnthropicChatMessageContent"/> class
     /// </summary>
-    /// <param name="calledToolResult">The result of tool called by the kernel.</param>
-    public AnthropicChatMessageContent(AnthropicFunctionToolResult calledToolResult)
-        : base(
-            role: AuthorRole.Assistant,
-            content: null,
-            modelId: null,
-            innerContent: null,
-            encoding: Encoding.UTF8,
-            metadata: null)
-    {
-        Verify.NotNull(calledToolResult);
-
-        this.CalledToolResult = calledToolResult;
-    }
-
-    /// <summary>
-    /// Initializes a new instance of the <see cref="AnthropicChatMessageContent"/> class.
-    /// </summary>
-    /// <param name="role">Role of the author of the message</param>
-    /// <param name="content">Content of the message</param>
-    /// <param name="modelId">The model ID used to generate the content</param>
-    /// <param name="calledToolResult">The result of tool called by the kernel.</param>
-    /// <param name="metadata">Additional metadata</param>
-    internal AnthropicChatMessageContent(
-        AuthorRole role,
-        string? content,
-        string modelId,
-        AnthropicFunctionToolResult? calledToolResult = null,
-        AnthropicMetadata? metadata = null)
-        : base(
-            role: role,
-            content: content,
-            modelId: modelId,
-            innerContent: content,
-            encoding: Encoding.UTF8,
-            metadata: metadata)
-    {
-        this.CalledToolResult = calledToolResult;
-    }
+    [JsonConstructor]
+    internal AnthropicChatMessageContent() { }
 
     /// <summary>
     /// Initializes a new instance of the <see cref="AnthropicChatMessageContent"/> class.
     /// </summary>
     /// <param name="role">Role of the author of the message</param>
-    /// <param name="content">Content of the message</param>
+    /// <param name="items">Instance of <see cref="ChatMessageContentItemCollection"/> with content items</param>
     /// <param name="modelId">The model ID used to generate the content</param>
-    /// <param name="functionsToolCalls">Tool calls parts returned by model</param>
+    /// <param name="innerContent">Inner content object reference</param>
     /// <param name="metadata">Additional metadata</param>
     internal AnthropicChatMessageContent(
```

Comment:
WE are moving away of having a constructor with all the properties to a `init` property approach, you can abandon this constructor and set the properties in the initialization.